File: app/src/main/java/com/ryanrvldo/moviecatalogue/data/source/remote/response/ContentRatingResponse.java
package com.ryanrvldo.moviecatalogue.data.source.remote.response;

import com.google.gson.annotations.Expose;
import com.ryanrvldo.moviecatalogue.data.source.model.ContentRating;

import java.util.List;

import lombok.Data;

@Data
public class ContentRatingResponse {
    @Expose
    private List<ContentRating> results;
}

File: core/src/main/kotlin/com/ryanrvldo/core/data/source/remote/response/tvshows/ContentRatingResponse.kt
package com.ryanrvldo.core.data.source.remote.response.tvshows

import com.google.gson.annotations.SerializedName

data class ContentRatingResponse(
    @SerializedName("iso_3166_1") val iso: String?,
    @SerializedName("rating") val rating: String?,
)
